releasenotes/notes/fix-sanic-non-str-args-4d1b0f506bee3436.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fixed the handling of endpoint paths with non-string arguments,
```

Comment:
We should mention sanic here and it seems like the sentence is incomplete?

Reply:
Yes, good catch! I was convinced I specified that, but it must have been in the commit message! 😞